NEWS.md

```diff
@@ -1321,6 +1321,8 @@ Deprecated or removed
 
   * `ndigits(n, b, [pad])` is deprecated in favor of `ndigits(n, base=b, pad=pad)` ([#27908]).
 
+  * `srand` is deprecated in favor of the unexported `Random.seed` ([#27726]).
```

Comment:
I believe the PR number should be mentioned here

Reply:
The issue number is sufficient.